i18n/backends/database/database.go

```diff
@@ -26,8 +36,18 @@ func (backend *Backend) LoadTranslations() []*i18n.Translation {
 	return translations
 }
 
-func (backend *Backend) SaveTranslation(t *i18n.Translation) {
+func (backend *Backend) SaveTranslation(t *i18n.Translation) (err error) {
+	if utf8.RuneCountInString(t.Key) > 190 {
```

Comment:
This (and `20`, on R44 below), shouldn't be duplicated with the struct definition? Is it possible in Go to define the value "190" is a constant that is also used in the struct tag?